packages/esm-patient-chart-app/src/visit/visit-form/visit-form.test.tsx

```diff
@@ -126,8 +130,12 @@ jest.mock('../hooks/useVisitAttributeType', () => ({
   })),
 }));
 
-jest.mock('../hooks/useDefaultLocation', () => {
-  const requireActual = jest.requireActual('../hooks/useDefaultLocation');
+jest.mock('../hooks/useEmrConfiguration', () => ({
+  useEmrConfiguration: jest.fn(() => ({})),
+}));
```

Comment:
According to best practices, we shouldn't do this anymore because the "jest.mocked(useEmrConfiguration)" should handle this... but it doesn't, and in mocked objects in this file it appears to both use the "jest.mocked" pattern and then use the jest.mock('../hooks...')... if someone other than me can understand what is going on here, happy to fix.

Reply:
We don't need [partial mocks](https://o3-docs.openmrs.org/docs/frontend-modules/unit-and-integration-testing#you-should-not-need-to-use-partial-mocks) for hooks from core anymore, but we still need them for hooks defined within an app. 

For core hooks, I think the magic comes from these places: [jest.config.js](https://github.com/openmrs/openmrs-esm-core/blob/f1d8002b89cfe042580e6825b4b4d7def3ba1628/packages/apps/esm-devtools-app/jest.config.js), [esm-core's mock.tsx](https://github.com/openmrs/openmrs-esm-core/blob/8d4612d384f000990303365e0d8575ebf382bb4f/packages/framework/esm-framework/mock.tsx) and [esm-react-utils's mock.tsx](https://github.com/openmrs/openmrs-esm-core/blob/0120ab0a384d7a2adcd90aec88dc224802af6ab2/packages/framework/esm-react-utils/mock.tsx). It'd be nice if we have something similar for individual apps, or at least a standardized file to add partial mocks to so we don't have to do that in every *test.tsx file

Reply:
Thanks!  So just to confirm, the takeaway here is what I did is fine, correct?

Reply:
Yeah, I think so.